### PR TITLE
Update Docker Desktop setup and separate out MiniKube setup

### PR DESCRIPTION
**Pull request by mrutkows**
This PR seeks to separate out the MiniKube setup which somehow appears to have been placed under the Docker Desktop setup instructions.  While doing this I thought to update the instructions to be more accurate relative to the Docker Desktop preferences and menuing system.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

### CODE REVIEW

**Comment by mrutkows** [comment]
Please note that I did not intend to fix the Minikube instructions (as I do not use it); however, it appears that recently some quite extensive (and perhaps overkill as the pre-req development tools) instructions were added to `tutorial.md`:
https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md

which seem to me to be more suited for DEVELOPMENT.md ...

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -95,32 +95,38 @@ The recommended configuration is:
...
-1.  Click the Apply and Restart button to save the preferences.
+1.  Click the `Apply and Restart` button to save the preferences.
 
 1.  Switch the proper `kubectl` config context:
 
     ```bash
     kubectl config get-contexts # You should see docker-for-desktop in the previous command output
     kubectl config use-context docker-for-desktop
     ```
+    * *Note: Docker Desktop menu provides a `Kubernetes` menuitem that allows you to select between contexts which is equivalent to the `kubectl` command.*
```
Comment (pritidesai) [inline DEVELOPMENT.md:129]: NIT: s/menuitem/menu item/

**Pull request merged**